bears/python/requirements/PySafetyBear.py

```diff
@@ -11,21 +12,34 @@
 from coalib.settings.Setting import typed_list
 
 
-def cve_key_checker(vulnerability):
-    if 'cve' in vulnerability.data:
-        if vulnerability.data['cve'] is None:
-            return None
-        else:
-            return True
-    else:
-        return None
+# It was for old versions of safety and those versions will be allow in future.
+# def cve_key_checker(vulnerability):
```

Comment:
Keep it as it was, uncommented. Sorry if i was confusing.

Use pragma nocover for now.